lib/zendesk_apps_support/package.rb

```diff
@@ -31,19 +31,22 @@ def initialize(dir, is_cached = true)
     def validate(marketplace: true, skip_marketplace_translations: false)
       errors = []
       errors << Validations::Manifest.call(self)
+
       if has_valid_manifest?(errors)
         errors << Validations::Marketplace.call(self) if marketplace
         errors << Validations::Source.call(self)
         errors << Validations::Translations.call(self, skip_marketplace_translations: skip_marketplace_translations)
         errors << Validations::Requirements.call(self)
 
-        unless manifest.requirements_only? || manifest.marketing_only? || manifest.iframe_only?
+        unless requirements_marketing_iframe_only_app?(manifest)
```

Comment:
`if v1_app`?

Reply:
Not sure I understand this suggestion. If it's a `requirements_only` or `marketing_only` app it can still be "V2" right?

Agree that `requirements_marketing_iframe_only_app` isn't the most catchy method name, but I prefer that it's explicit